requirements.txt

```diff
@@ -2,3 +2,4 @@ anymarkup
 jsonpointer
 requests
 websocket-client
+GitPython
```

Comment:
Another alternative is to just only support generation from a local directory and  have the user run `git clone` before doing a "generation".